index.html

```diff
@@ -1,10 +1,194 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title></title>
-  </head>
-  <body>
 
-  </body>
+<head>
+  <meta charset="utf-8">
+  <title>Popular Github</title>
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
+   <link rel="stylesheet" href="./app.css">
+</head>
+
+<body>
+    <nav class="nav-bar">
+
+
+          <h1 class="nav-group">Shaun Willis</h1>
+
+          <ul class="nav-links">
+            <li class="nav-group__item"><a href="#">Home</a></li>
+            <li class="nav-group__item"><a href="#">About</a></li>
+          </ul>
```

Comment:
Look at using the "Beautify" command in atom to make this easier to read in terms of whitespace.